### PR TITLE
Document component variables

### PR DESCRIPTION
**Pull request by epetrow**
closes: https://github.com/telerik/kendo-themes-private/issues/125

### CODE REVIEW

File: packages/default/docs/customization-treeview.md [hunk 1]
```diff
@@ -303,7 +423,7 @@ The following table lists the available variables for customization.
     <td><span class="color-preview" style="background-color: #d6534a"></span><code>#d6534a</code></td>
 </tr>
 <tr>
-    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">Text color of a hovered load more.</div></div>
+    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The text color of the Treeviews' hovered load more button.</div></div>
```
Comment (yordan-mitev) [inline packages/default/docs/customization-treeview.md:426]: Edit as shown above.

File: packages/default/docs/customization-treeview.md [hunk 2]
```diff
@@ -323,7 +443,7 @@ The following table lists the available variables for customization.
     <td><span class="color-preview" style="background-color: transparent"></span><code>transparent</code></td>
 </tr>
 <tr>
-    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">Background color of a focused load more.</div></div>
+    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The background color of the Treeviews' focused load more button.</div></div>
```
Comment (yordan-mitev) [inline packages/default/docs/customization-treeview.md:446]: Edit as shown above.

File: packages/default/docs/customization-treeview.md [hunk 3]
```diff
@@ -333,7 +453,7 @@ The following table lists the available variables for customization.
     <td><span class="color-preview" style="background-color: #d6534a"></span><code>#d6534a</code></td>
 </tr>
 <tr>
-    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">Text color of a focused load more.</div></div>
+    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The text color of the Treeviews' focused load more button.</div></div>
```
Comment (yordan-mitev) [inline packages/default/docs/customization-treeview.md:456]: Edit as shown above.

File: packages/default/docs/customization-treeview.md [hunk 4]
```diff
@@ -343,7 +463,7 @@ The following table lists the available variables for customization.
     <td><code>null</code></td>
 </tr>
 <tr>
-    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">Border color of a focused load more.</div></div>
+    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The border color of the Treeviews' focused load more button.</div></div>
```
Comment (yordan-mitev) [inline packages/default/docs/customization-treeview.md:466]: Edit as shown above.

File: packages/default/docs/customization-treeview.md [hunk 5]
```diff
@@ -353,7 +473,7 @@ The following table lists the available variables for customization.
     <td><code>inset 0 0 0 2px rgba(0, 0, 0, 0.13)</code></td>
 </tr>
 <tr>
-    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">Box shadow of a focused load more.</div></div>
+    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The box shadow of the Treeviews' focused load more button.</div></div>
```
Comment (yordan-mitev) [inline packages/default/docs/customization-treeview.md:476]: Edit as shown above.

File: packages/default/docs/customization.md [hunk 6]
```diff
@@ -491,6 +491,16 @@ The following table lists the available variables for customizing the Default th
     <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">Opacity used for disabled items.</div></div>
     </td>
 </tr>
+<tr>
+    <td>$kendo-floating-label-font-size</td>
```
Comment (yordan-mitev) [inline packages/default/docs/customization.md:495]: monospace

File: packages/default/docs/customization.md [hunk 7]
```diff
@@ -491,6 +491,16 @@ The following table lists the available variables for customizing the Default th
...
+    <td>Number</td>
+    <td><code>$kendo-input-font-size</code></td>
+    <td><code>14px</code></td>
+</tr>
+<tr>
+    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The font size of the Floating label.<br /> @group floating-label</div></div>
```
Comment (yordan-mitev) [inline packages/default/docs/customization.md:501]:
```suggestion
    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The font size of the Floating Label.<br /> @group floating-label</div></div>
```

File: packages/default/docs/customization.md [hunk 8]
```diff
@@ -652,7 +752,37 @@ The following table lists the available variables for customizing the Default th
     <td><code>1</code></td>
 </tr>
 <tr>
-    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">Line heights used along with $kendo-font-size.</div></div>
+    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The line heights used along with $kendo-font-size.</div></div>
```
Comment (yordan-mitev) [inline packages/default/docs/customization.md:755]:
```suggestion
    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The line heights used along with `$kendo-font-size`.</div></div>
```

File: packages/default/docs/customization.md [hunk 9]
```diff
@@ -632,7 +672,37 @@ The following table lists the available variables for customizing the Default th
...
+    </td>
+</tr>
+<tr>
+    <td>$kendo-badge-sm-padding-y</td>
+    <td>Number</td>
+    <td><code>k-map-get( $kendo-spacing, 0.5 )</code></td>
+    <td><code>2px</code></td>
+</tr>
+<tr>
+    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The vertical padding of the small Badge.</div></div>
```
Comment (yordan-mitev) [inline packages/default/docs/customization.md:685]: In all descriptions that follow, apply the suggestions from the individual component files above.

File: packages/default/scss/badge/_variables.scss [hunk 10]
```diff
@@ -1,50 +1,80 @@
...
 $kendo-badge-md-line-height: $kendo-badge-line-height !default;
+/// The line height used along with $kendo-font-size of the large Badge.
+/// @group badge
 $kendo-badge-lg-line-height: $kendo-badge-line-height !default;
 
-/// Badge min width used for circle badge.
+/// The calculated minimum width of the Badge used for circular Badge.
 /// @group badge
 $kendo-badge-min-width: calc( #{$kendo-badge-line-height * 1em} + #{$kendo-badge-padding-y * 2} + #{$kendo-badge-border-width * 2}) !default;
+/// The calculated minimum width of the small Badge used for circular Badge.
```
Comment (yordan-mitev) [inline packages/default/scss/badge/_variables.scss:67]: As these descriptions repeat those in the `docs` folder, apply the same suggestions.

**Review by Juveniel — APPROVED** [review]

**Pull request merged**